mandatory/4-tax.js

```diff
@@ -5,7 +5,13 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax() {
+  let salesTax = 0.2;
+  let totalProductPrice = productPrice + productPrice * 0.2;
+  return total;
```

Comment:
hi saliha, i liked the way  you doing it but i don't see your total variable is defined anywhere.This is another way of doing it:
function calculateSalesTax() {
  const tax = 0.2 * sales;
  return tax + sales;
}